ethereum/core/src/main/java/org/hyperledger/besu/ethereum/transaction/CallParameter.java

```diff
@@ -28,7 +28,7 @@
 
   private final Address to;
 
-  private final long gasLimit;
+  private long gasLimit;
```

Comment:
See my comment on clone and copy.  Making this the only mutable field seems awkward.